netmaster/objApi/apiController.go

```diff
@@ -880,6 +882,6 @@ func (ac *APIController) BgpDelete(bgpNeighborCfg *contivModel.Bgp) error {
 }
 
 //BgpUpdate updates bgp config
-func (ac *APIController) BgpUpdate(oldbgpNeighborCfg *contivModel.Bgp, NewbgpNeighborCfg *contivModel.Bgp) error {
+func (ac *APIController) BgpUpdate(oldbgpCfg *contivModel.Bgp, NewbgpCfg *contivModel.Bgp) error {
 	return nil
```

Comment:
This is a filler function. Currently not being used. I believe this was added to keep compiler from not complaining about non implementation of the function